lib/propolis/src/block/in_memory.rs

```diff
@@ -8,11 +8,17 @@ use std::sync::{Arc, Mutex};
 
 use crate::accessors::MemAccessor;
 use crate::block;
+use crate::migrate::{
+    MigrateCtx, MigrateSingle, MigrateStateError, Migrator, PayloadOffer,
+    PayloadOutput,
+};
 use crate::tasks::ThreadGroup;
 use crate::vmm::{MemCtx, SubMapping};
 
 use anyhow::Context;
 
+use super::Lifecycle;
```

Comment:
style nit: use this out of `crate::common`, rather than the re-export from `block`